src/test/java/com/onaio/steps/handler/actions/NotReachableHandlerTest.java

```diff
@@ -43,7 +44,7 @@
     private Household householdMock;
     private NotReachableHandler notReachableHandler;
     private CustomDialog dialogMock;
-    private int MENU_ID = R.id.action_not_reachable;
+    private final int MENU_ID = R.id.action_not_reachable;
```

Comment:
This is a constant so change it to `final static`

Reply:
done